main_2024.py

```diff
@@ -160,12 +171,8 @@ def main() -> int:
             print("pitch: " + str(orientation.pitch))
             print("")
 
-        if image is None:
-            continue
-
-        cv2.imshow("Landing Pad Detector", image)
-
-        if cv2.waitKey(1) & 0xFF == ord('q'):
+        if cv2.waitKey(1) == ord("q"):
```

Comment:
Single quotations around `q` since input to `ord()` is always a single character.